tests/drivers/dma/chan_blen_transfer/src/test_dma.c

```diff
@@ -26,8 +26,16 @@
 #define DMA_DEVICE_NAME CONFIG_DMA_0_NAME
 #define RX_BUFF_SIZE (48)
 
+#ifdef CONFIG_NOCACHE_MEMORY
```

Comment:
please move all the changes in `tests/` to a separate commit

Reply:
ok